graphdatascience/resources/imdb/serialize_imdb.py

```diff
@@ -0,0 +1,68 @@
+from importlib.resources import path
```

Comment:
This file should be pruned out of packaging in `MANIFEST.in`

Reply:
The path to `serialize_cora.py` there also needs to be updated now

Reply:
👍  didn't know about this, that's good that we keep the package clean. Added now.